docs/advanced-features/using-mdx.md

```diff
@@ -98,6 +119,26 @@ author: 'Rich Haines'
 # My MDX page
 ```
 
+```js
```

Comment:
This code block shows how to add remarkFrontmatter but the line above it shows how to add metadata using `export const meta` which does not require any extra configuration. Maybe this can be in it's own section?